modules/virt-creating-a-custom-image-boot-source-web.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.adoc
+
+[id="virt-creating-a-custom-disk-image-boot-source-web_{context}"]
+= Creating a custom boot source
+
+Sometimes you want to prepare a custom disk image, based on an existing disk image, for use as a boot source.
+
+This procedure guides you through:
+
+* Preparing a custom disk image.
+* Creating a boot source from the custom disk image.
+* Attaching the boot source to a custom template.
+
+.Procedure
+. In the {VirtProductName} console, click *Workloads* > *Virtualization* from the side menu.
+. Click the *Templates* tab.
+. Click the link in the *Source provider* column for the template you want to customize. A window displays, indicating that the template currently has a *defined* _source_.
+. In the window, click the *Customize source* link.
+. In the *About boot source customization* window, click *Continue* to proceed with customization, after reading the information provided about the boot source customization process.
+. On the *Prepare boot source customization* page, in the *Define new template* section:
+.. Select the *New template namespace* field and choose *PR default*.
```

Comment:
It hasn't to be 'default' and `PR` is an abbreviation. Suggestion:
Click the *New template namespace* field and select a project.

Reply:
Thanks @gouyang. Before I make this change, can you also confirm that labels (in **Bold**) in these steps:

- _Verify the custom image works as expected. If it does, click **Boot source available**._
- 
- _Click **Start Customization**. The customization process begins and the Preparing boot source customization page displays, followed by the Customize boot source page. The Customize boot source page displays the output of the running script. When the script completes, your custom image is created._

"Boot source available" doesn't seem right to me, and I had a hard time seeing the label in the video. Just want to be sure, as I can't access a build.

Reply:
You're right, it's `Make this boot source available`.

Reply:
Great! Thanks, fixed.

Reply:
I'm not sure whether it should be click or select on the drop-downs, but peer review will set me straight on that. Changing to: Select the *New template namespace* field and choose a project.